jobs/acceptance-tests/templates/test.erb

```diff
@@ -115,4 +115,7 @@ done
 
 export GOFLAGS=-mod=vendor
 cd /var/vcap/packages/acceptance-tests/src/github.com/cloudfoundry/cf-acceptance-tests || exit 1
-exec bin/test
+exec bin/test \
+  --noisySkippings=<%= p('acceptance_tests.ginkgo.noisy_skippings') %> \
+  --flakeAttempts=<%= p('acceptance_tests.ginkgo.flake_attempts') %> \
+  --slowSpecThreshold=<%= p('acceptance_tests.ginkgo.slow_spec_threshold') %>
```

Comment:
Since you're here, could we also add `"${@}"` here? That  would mean we _could_ edit the ejob directly for quick tests.